ingredient/ingredient-arm/src/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "module": "commonjs",
     "declaration": true,
     "outDir": "../dist",
-    "strict": true,
+    "strict": false,
```

Comment:
Any reason for removing strict?

Reply:
I removed strict because I was troubleshooting typing errors between packages. Since those have been resolved I will return those to true to prevent the introduction of typing issues.